developer-portal/excluded-docs/_TEMPLATE-PLATFORM_NAME-fast-checkout-on-amp-pages.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Fast PLATFORM_NAME Docs: Fast Headless Checkout on AMP Sites with PLATFORM_NAME"
+description: Fast Headless Checkout on AMP Sites with PLATFORM_NAME
+keywords: Fast Headless Checkout AMP PLATFORM_NAME
+exclude: true
+---
+
+# Using Fast Headless Checkout on AMP Sites with PLATFORM_NAME
+
+<embed src="/reusables/for-developers/_platform_headless_all_amp_banner_info_about_amp.md" />
+
+## Restrictions
+
+<embed src="/reusables/for-developers/_platform_headless_all_amp_banner_additional_future_requirements.md" />
+
+The following must be true in order for you to use Fast Checkout on an AMP page:
+
+<embed src="/reusables/for-developers/_platform_headless_all_requirement_catalog_integration.md" />
```

Comment:
Little confused by the wording here. I'd expect a proper list to exist after "the following must be true"